planemo_ext/galaxy/tools/linters/xml_order.py

```diff
@@ -17,16 +17,37 @@
     'citations',
 ]
 
+DATASOURCE_TAG_ORDER = [
+    'description',
+    'command',
+    'inputs',
+    'request_param_translation',
+    'uihints',
```

Comment:
That's an awful good question regarding uihints.

Yes, sorry, I removed everything I didn't see during very brief greps through the tools directory.